lib/screens/authenticate/sign_in.dart

```diff
@@ -60,12 +68,26 @@ class _SignInState extends State<SignIn> {
                       ),
                       SizedBox(height: 20.0),
                       TextFormField(
-                        decoration:
-                            textInputDecoration.copyWith(hintText: 'Password'),
+                        decoration: textInputDecoration.copyWith(
+                          hintText: 'Password',
+                          suffixIcon: IconButton(
+                            icon: Icon(
+                              passwordHide
+                                  ? Icons.visibility_off
+                                  : Icons.visibility,
```

Comment:
Although not a big issue, clicking on IconButton ripples hide behind the textbox. It should be in front.